tasks/main.yml

```diff
@@ -4,6 +4,12 @@
   with_items:
   - git
 
+- name: Install socat
+  package:
+    name: socat
+    state: present
+  when: enable_onion_support is defined
```

Comment:
Yeah, that's a part where I have doubt, because we use it later the same trick for git_shell. But if that's too magic, better change it.